src/components/profile/components/Milestones.tsx

```diff
@@ -1,121 +1,127 @@
-import React from "react";
-import twitter from '../../../assets/images/X.com.png'
-import eightyseven from '../../../assets/images/87percent.png'
-import ten from '../../../assets/images/10percent.png'
+import React from 'react'
+import { ReactComponent as Twitter } from '../../../assets/dashboard/XLogo.svg'
 
+interface ListItemProps {
+  icon: JSX.Element
+  text: string
+  xp: number
+  progress?: number
+  opacity?: boolean
+}
 
-const CircularProgress = ({ percentage }) => {
-    const radius = 45;
-    const circumference = 2 * Math.PI * radius;
-    const strokeDasharray = circumference;
-    const strokeDashoffset = circumference - (percentage / 100) * circumference;
-
-    return (
-        <svg className="w-12 h-12" viewBox="0 0 100 100">
-            {/* Background circle */}
-            <circle
-                className="text-gray-200"
-                strokeWidth="10"
-                stroke="currentColor"
-                fill="transparent"
-                r={radius}
-                cx="50"
-                cy="50"
-            />
-            {/* Progress circle */}
-            <circle
-                className="text-[#293056]"
-                strokeWidth="10"
-                strokeLinecap="round"
-                stroke="currentColor"
-                fill="transparent"
-                r={radius}
-                cx="50"
-                cy="50"
-                strokeDasharray={strokeDasharray}
-                strokeDashoffset={strokeDashoffset}
-                style={{
-                    transition: 'stroke-dashoffset 0.5s ease',
-                    transform: 'rotate(-90deg)',
-                    transformOrigin: '50% 50%',
-                }}
-            />
-            {/* Percentage text in the middle */}
-            <text
-                x="50%"
-                y="50%"
-                dominantBaseline="middle"
-                textAnchor="middle"
-                fontSize="25"
-                className="font-semibold text-black"
-            >
-                {percentage}%
-            </text>
-        </svg>
-    );
-};
+const ListItem: React.FC<ListItemProps> = ({

```

Comment:
Everywhere we are using the function/component declaration like this.
`const ListItem = ({ destructureProps }: ListItemProps) {...}`

Then why are you using this `React.FC<type>` declaration? Any specific reason?